__tests__/client.test.js

```diff
@@ -62,6 +62,11 @@ describe("generator-engage:client", () => {
       expect(json.dependencies["webpack-cli"]).toBeDefined();
```

Comment:
While i was in there I noticed this -- should it be checking devDependencies/have it there, or should the description just say "dependency"? This is conflicting

Reply:
dependencies in react are confusing. Everything gets bundled into one large js file anyway, but webpack-cli is required in the client to build in production.